### PR TITLE
Add persp-next and persp-prev functions

### PR DESCRIPTION
**Pull request by CestDiego**
just like in perspectives.el

### CODE REVIEW

**Comment by CestDiego** [comment]
@Bad-ptr just as a side-note, is there a way to `blacklist` a perspective so it's not saved? For example I have a custom perspective for ERC, and when I restart Emacs the perspective just stays there without connecting.

**Comment by Bad-ptr** [comment]
> is there a way to blacklist a perspective so it's not saved?

No, but it could be implemented.

**Comment by CestDiego** [comment]
How could it be implemented? Any hints?

File: persp-mode.el [hunk 1]
```diff
@@ -862,6 +869,35 @@ instead it's contents will be erased.")
...
+     ((null pos) nil)
+     (only-perspective? nil)
+     ((= pos (1- persp-list-length))
+      (if persp-switch-wrap (persp-switch (nth 0 persp-list))))
+     (t (persp-switch (nth (1+ pos) persp-list))))))
+
+(defun persp-prev ()
+  "Switch to previous perspective (to the left)."
+  (interactive)
+  (let* ((persp-list (persp-names-current-frame-fast-ordered))
```
Comment (CestDiego) [inline persp-mode.el]: @Bad-ptr I made the changes. The function is a little too long but I guess it's ok

**Comment by Bad-ptr** [comment]
I was near to press the 'merge pull request' button, but then decided to take a closer look...)
In the persp-next you use `cl-position`, and in the persp-prev -- `position`. As persp-mode currently uses cl (not cl-lib), the right function to use is `position`.(Am I right?)

**Comment by Bad-ptr** [comment]
> is there a way to blacklist a perspective so it's not saved?
>> How could it be implemented? Any hints?

The hint:
I think you cold set something like `dont-save-to-file` perspective parameter(see `set-persp-parameter` and `persp-parameter` functions) and filter out perspectives which have this parameter set to `t` in the `persps-to-savelist` function.

**Comment by CestDiego** [comment]
> In the persp-next you use cl-position

This was a typo. I fixed it

**Pull request merged**